crates/harness-tests/src/routing.rs

```diff
@@ -199,3 +201,36 @@ async fn monolith_double_load_room(ctx: &mut TestRunner) {
     let t = resp.text().await.expect("failed to read http response");
     assert_eq!(t, "{\"name\":\"foo\"}");
 }
+
+#[test_context(TestRunner)]
+#[tokio::test]
+async fn unicast_messaging(ctx: &mut TestRunner) {
+    let mut m = MonolithBuilder::new()
+        .behavior(BehaviorTrackClients)
+        .build(ctx)
+        .await;
+
+    m.show().await;
+
+    let mut c1 = Client::new(ctx).unwrap();
+    c1.join("foo").await;
+    let mut c2 = Client::new(ctx).unwrap();
+    c2.join("foo").await;
+
+    m.wait_recv().await;
+
+    let c_id = m.clients().iter().next().copied();
```

Comment:
HashSet does not maintain the order of items that are put in it, so this is not guarenteed to always get the client id of `c1`.